staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -48,3 +48,89 @@ type AdmissionPluginConfiguration struct {
 	// +optional
 	Configuration *runtime.Unknown
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Webhook describes a webhook and the resources and operations it applies to.
+type Webhook struct {
```

Comment:
I am confused about why this is a separate type from the backend configuration.

Reply:
yeah this may not be necessary, just wrapping the client configuration similar to the current admission package. We can scrap it